### ISSUE TITLE
Inclusion of code lists used for elements/attributes external to IWXXM

### ISSUE THREAD

**Issue by blchoy** [wmo-im/iwxxm#193]
The Python script codeListsToSchematron.py used to download RDF files from the WMO Codes Registry to support validation of code list entries in IWXXM instances is not able to download the "nil" code list as it was not mentioned in the IWXXM XSDs.  This is because this code list is being used in @nilReason which is an attribute defined external to IWXXM.

For completeness, it is desirable to also include the code list used by external element/attribute in common.xsd.  This will not affect representation of the reports but can give a more complete picture of the model.

**Comment by blchoy** [comment]
For information, relevant section in an XSD for a code list is as below:

```
<complexType name="CloudAmountReportedAtAerodromeType">
	<annotation>
		<documentation>...</documentation>
		<appinfo>
			<vocabulary>http://codes.wmo.int-49-2/CloudAmountReportedAtAerodrome</vocabulary>
			<extensibility>none</extensibility>
		</appinfo>
	</annotation>
	<complexContent>
		<extension base="gml:ReferenceType"></extension>
	</complexContent>
</complexType>
```

**Comment by blchoy** [comment]
This has already been fixed in IWXXM 3.0.0RC4.
